testing/trino-testing-containers/src/main/java/io/trino/testing/containers/Alluxio.java

```diff
@@ -43,8 +47,16 @@ public Alluxio()
 
     public void start()
     {
-        alluxioMaster.start();
-        alluxioWorker.start();
+        Failsafe.with(RetryPolicy.<Void>builder()
+                        .withMaxAttempts(10)
```

Comment:
Given startup timeout is 3m, 10 attempts is looong time. Would it work to limit it to say 10 minutes?